src/evergreen/performance_results.py

```diff
@@ -0,0 +1,176 @@
+# -*- encoding: utf-8 -*-
+"""Performance results representation of evergreen."""
+from __future__ import absolute_import
+
+import copy
+
+from evergreen.base import _BaseEvergreenObject, evg_attrib, evg_short_datetime_attrib, \
+    evg_datetime_attrib
+from evergreen.util import parse_evergreen_datetime
+
+
+class TestResult(_BaseEvergreenObject):
```

Comment:
[nit] Everything here that is prefixed with `Test` might be better prefixed with `PerfTest` since these are only applicable to performance tests.